azure-pipelines.yml

```diff
@@ -94,7 +94,16 @@ stages:
   - template: ${{ variables.releaseTemplate }}
     parameters:
       environment: 'test'
-  - job: 
+  - job:
+    displayName: 'Load Testing'
+    steps:
+      - task: k6-load-test@0
+        inputs:
+          filename: 'test/k6/ci/status/get-status.js'
+          args: '-e api_base_url=https://app-opdex-test-api-staging.azurewebsites.net'
+        displayName: 'GET /status'
+    dependsOn: 'test' # template deployment name == parameters.environment
+  - job:
```

Comment:
We can try it but tbh I'm not a big fan of load or integration tests in CI/CD pipelines due to added length of deployments. In emergencies we want to be fast, waiting 10-20 minutes for a deployment could be a problem.

Reply:
Yeah I think you make a good point. Let's keep this single test, it will still be useful as a health check and I see it being useful when attempting to automate deployment slot swaps